### DISCUSSION TITLE
Big Sur typing delay

### DISCUSSION THREAD

**Discussion by KathrynCrawford** [atom/atom#23046] [Q&A]
I recently updated to Big Sur (11.5.2) and am having major typing delays in Atom. So far it only seems to effect Atom, and the issue happens with the external and internal keyboards. Also doesn't seem to be related to autocomplete, as I disabled it and had the same issue.
![Sep-28-2021 11-01-32](https://user-images.githubusercontent.com/10589544/135125645-4ed90907-d025-433e-8263-26f013d9c96c.gif)

**Comment by Tijil2111** [comment]
Do you have a M1 chip or an intel processor ?

**Reply by KathrynCrawford** [reply]
Intel

**Reply by ThatXliner** [reply]
I cannot reproduce

EDIT: actually, I’m not sure which specific version of Big Sur I have. Will check later

**Reply by Tijil2111** [reply]
> Intel

Oh I thought that might be specific to M1 because Atom isnt ready for M1 yet although Rosetta 2 works. Then sorry I cant help. Maybe some OS related issue. Who knows

**Reply by Tijil2111** [reply]
Hey Maybe check this 
https://stackoverflow.com/questions/45461641/atom-editor-running-slowly-in-all-new-projects

Maybe this works
